example_tpv_config_locations_api.yml

```
@@ -20,14 +20,14 @@ tools:
       import pathlib
       from ruamel.yaml import YAML
 
-      # NOTE: currently object store info is stored in a yaml 
+      # NOTE: currently object store info is stored in a yaml
       objectstore_loc_path = "tests/fixtures/object_store_locations.yml"
-      selected_object_store = "object_store_australia"
+      dataset_attributes = helpers.get_dataset_attributes(job.get_input_datasets())
```

Correct me if I'm wrong but his seems to be used like this: 
```suggestion
      dataset_attributes = helpers.get_dataset_attributes(job.input_datasets)
```

---

Both should work. However, we can switch it to whatever is in the TPV.